src/lib/staged_ledger/staged_ledger.ml

```diff
@@ -6,6 +6,7 @@ open Currency
 open Signature_lib
 module Ledger = Mina_ledger.Ledger
 module Sparse_ledger = Mina_ledger.Sparse_ledger
+module Check_commands = Check_commands
```

Comment:
Let's remove it and instead add

```
type transaction_pool_proxy = Check_commands.transaction_pool_proxy
```

because there is nothing else of interest in the module for export